### PR TITLE
Add Storage Concealment Implant

### PR DESCRIPTION
**Pull request by LeDrascol**
# About The Pull Request
Adds a new implant that hides your back-slot item. The implant can be toggled on and off with a button, which plays a "fake spark" effect and displays a chat message.

The item is obtained from the same techweb node and protolathe printing as other implants.

## Why It's Good For The Game
Many users have expressed a desire for "invisible backpacks" or the ability to hide the backpack. This method integrates it with science fiction technology.

Related to Discord suggestions 26, 3764, and 3672.

## A Port?
No.

## Pre-Merge Checklist
- [X] You tested this on a local server.
- [X] This code did not runtime during testing.
- [X] You documented all of your changes.

## Changelog
:cl:
add: Added the Storage Concealment Implant
/:cl:

### CODE REVIEW

File: modular_splurt/code/modules/mob/living/carbon/human/update_icons.dm [hunk 1]
```diff
@@ -0,0 +1,15 @@
+// Function for updating back sprites
+/mob/living/carbon/human/update_inv_back()
+	. = ..()
+	
+	// Check for hidden backpack trait
+	if(HAS_TRAIT(src, TRAIT_HIDE_BACKPACK))
+		..()
```
Comment (BongaTheProto) [inline modular_splurt/code/modules/mob/living/carbon/human/update_icons.dm:7]: Is it necessary to call `..()` twice in the same proc?

**Review by BongaTheProto — COMMENTED** [review]
So, since this is something players want for cosmetic reasons and doesn't give that much advantage, would it be possible to make a neutral quirk that gives you the implant on spawn?

File: modular_splurt/code/modules/mob/living/carbon/human/update_icons.dm (hunk 1 above)
Comment (LeDrascol) [reply modular_splurt/code/modules/mob/living/carbon/human/update_icons.dm:7]: This was an accidental inclusion. Further testing revealed it to be unnecessary.

**Comment by LeDrascol** [comment]
> would it be possible to make it a neutral quirk that gives you the implant on spawn?

This was created without an associated quirk to create new interaction opportunities for the science and medical departments. A quirk based on it could be added.

**Pull request merged**